index.html

```diff
@@ -179,8 +179,9 @@ <h2 class="sr-only">Learning Management System integrations</h2>
         </section>
     </div>
     <footer>
-        <div id="footer-video-container">
-            <iframe width='853' height='480' src="https://www.youtube.com/embed/PK-2DBIVDO8?&theme=dark&color=white&autohide=2&cc_load_policy=1&modestbranding=1&showinfo=0&rel=0"frameborder="0"></iframe>
+        <div class='embed-container'>
```

Comment:
Should we put a max-width on this container? When viewing full-width on my screen, it's pretty massive 

<img width="1792" alt="screen shot 2017-02-06 at 14 23 13" src="https://cloud.githubusercontent.com/assets/109850/22648620/ceed1c8a-ec77-11e6-8ea6-d2f71ce07f9d.png">